### DISCUSSION TITLE
planning.py作业中grade.py的错误信息具有误导性

### DISCUSSION THREAD

**Discussion by yuanyi-thu** [physics-data/faq#56] [General] [answered]
grade.py中子进程超时之后只是将变量message的值设为了超时提示，而没有输出message，随后程序立即杀死了子进程
而此时超时的子进程并没有输出文件
进入else部分后程序会发现这一问题，并将message改为”没有找到输出文件“并最终输出
但是真正的问题是程序的超时
这一错误信息有较大的误导性，建议改为判定超时之后直接输出超时提示

**Comment by Harry-Chen** [comment, marked as answer]
超时导致被杀死导致没有输出文件，其实也是正确的。我们会考虑在后面的 grader 里面增加超时的警告
